pyfr/readers/native.py

```diff
@@ -165,6 +165,21 @@ def __iter__(self):
     def __len__(self):
         return len(self._npf.files)
 
+class PyFRH5Reader(PyFRBaseReader):
+    """Does everything that file reader does but with a h5py file"""
+    def __init__(self,fname):
+        import h5py
+        self._file = h5py.File(fname,'r')
+
+    def __getitem__(self,aname):
+        return np.array(self._file[aname])
+
+    def __iter__(self):
+        return iter(self._file.keys())
```

Comment:
Just checked:
```
In [10]: f.keys()
Out[10]: [u'a', u'b']

In [11]: list(f)
Out[11]: [u'a', u'b']
```
Still might need the `iter()`, though, but the `.keys()` can go.